helm/higress/templates/configmap.yaml

```diff
@@ -115,4 +115,16 @@ data:
 {{- include "mesh" . }}
 {{- end }}
 ---
-
+  {{- if .Values.gateway.customBootstrap.enabled  }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: higress-custom-bootstrap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gateway.labels" . | nindent 4 }}
+data:
+  custom_bootstrap.json:
+    "{\"stats_sinks\":[{\"name\":\"envoy.metrics_service\",\"typed_config\":{\"@type\":\"type.googleapis.com/envoy.config.metrics.v3.MetricsServiceConfig\",\"transport_api_version\":\"V3\",\"grpc_service\":{\"envoy_grpc\":{\"cluster_name\":\"service_skywalking\"}}}}],\"static_resources\":{\"clusters\":[{\"name\":\"service_skywalking\",\"type\":\"LOGICAL_DNS\",\"connect_timeout\":\"5s\",\"http2_protocol_options\":{},\"dns_lookup_family\":\"V4_ONLY\",\"lb_policy\":\"ROUND_ROBIN\",\"load_assignment\":{\"cluster_name\":\"service_skywalking\",\"endpoints\":[{\"lb_endpoints\":[{\"endpoint\":{\"address\":{\"socket_address\":{\"address\":\"{{ .Values.gateway.customBootstrap.skywalking.address }}\",\"port_value\":\"{{ .Values.gateway.customBootstrap.skywalking.port }}\"}}}}]}]}}]}}"
```

Comment:
This configuration is difficult to maintain, please use helm functions such as `toJson` to optimize it.

Reply:
ok